docs/components/concepts/job-workers.md

```diff
@@ -236,3 +236,31 @@ This returns the current view of the registered job streams, where `client` refe
 For example, if jobs of a given type are not activated, but a worker is opened for this type, you can verify first if it exists in one of the gateways as a client stream. Once you've found it, grab its ID, and verify that you can find it as a consumer of a remote stream on each broker.
 
 If it's not present in the gateway as a client stream, restart your worker. If it's not present as a consumer in one of the brokers, this indicates a bug. As a workaround, restart your gateway, which will cause some interruption in your service, but will force all streams for this gateway to be recreated properly.
+
+## Performance best-practices
```

Comment:
❓ Wondering whether we either should move this out or at least link it from somewhere else like reference/best practices guide or something?